mapillary_tools/upload.py

```diff
@@ -66,7 +66,7 @@ def wrap_http_exception(ex: requests.HTTPError):
     lines = [
         f"{ex.request.method} {resp.url}",
         f"> HTTP Status: {ex.response.status_code}",
-        f"{resp.content}",
+        f"{resp.content.decode()}",
```

Comment:
It's intentional to be `resp.content` without decoding, because it could be binary that are not decodable.

If it fails the type check, try `f"{str(resp.content)}"` instead